vme/zone/udgaard.zon

```diff
@@ -11246,7 +11242,7 @@ code
    if (not ((NICHOLAS_QUEST+" Complete") in activator.quests))
       goto start;
 
-   if (ReadyCareer@professions(activator))
```

Comment:
This looks wrong, I think it must be ReadyCareer(), otherwise you move the logic onto the teachers